modules/displaybook.js

```diff
@@ -0,0 +1,65 @@
+const bookContainer = document.querySelector('.books-container');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+
+let bookList = [];
+
+const bookUpload = () => {
+  bookContainer.innerHTML = '';
+  bookList.forEach((books) => {
+    bookContainer.innerHTML += `
+      <div class="books">
+        <div class ="book-wrapper"> 
+          <div class="book-holder">
+              <h3>"${books.title}"</h3>
+              <h3>by</h3>
+              <h3>${books.author}</h3>
+          </div>   
+          <button class= "delete-btn" data-id="${books.id}">Remove</button>
+        </div>   
+          <hr />
+      </div>
+     `;
+    bookTitle.value = '';
+    bookAuthor.value = '';
+  });
+  const bookRemoveBtn = document.querySelectorAll('.delete-btn');
+
+  bookRemoveBtn.forEach((button) => {
+    button.addEventListener('click', () => {
+      const dataSet = parseInt(button.dataset.id, 10);
+      const buttonId = bookList.findIndex((object) => object.id === dataSet);
+      // eslint-disable-next-line no-use-before-define
+      deleted(buttonId);
```

Comment:
- [ ] [REQUIRED] Good job so far! In my opinion, you must have thought that the previous code [reviewer's requirement](https://github.com/shegy28/Awesome-Books-App-ES6/pull/1#discussion_r1039898706) is only to `disable linter` comment on `1` line 😃, please don't use any `disable linters` comment, it will lead you to lack experience, solve any problem by yourself and your skill level will increase even more 👍🏼

Reply:
@mukhammadrizooff  what am i supposed to do now